policy/policy_handler/proxy.go

```diff
@@ -7,15 +7,15 @@ import (
 	"github.com/atomist-skills/go-skill/policy/data/proxy"
 )
 
-func WithProxyClient(url string) Opt {
+func WithProxyClient() Opt {
 	return func(h *EventHandler) {
-		provider := getProxyClientProvider(url)
```

Comment:
Not urgent, but we could unwrap this func since the signatures are now the same.

I think this line could just be something like `provider := proxy.NewProxyClientFromSkillRequest`